libbeat/docs/processors-list.asciidoc

```diff
@@ -180,6 +187,14 @@ endif::[]
 ifndef::no_add_process_metadata_processor[]
 include::{libbeat-processors-dir}/add_process_metadata/docs/add_process_metadata.asciidoc[]
 endif::[]
+
+
+
+ifndef::no_add_session_metadata_processor[]
+* <<add-sessionmetadata,`add_session_metadata`>>
+endif::[]
+
```

Comment:
Note to sel @karenzone: Don't forget these extra returns. Leaving in for now for easy spotting.